Code releases 05.20/d8e84299-d99b-40e5-a34d-791101435ee6.md

```diff
@@ -1,5 +1,39 @@
+The super attribute is the only way to tell Spryker, that a product contains multiple variants (or concrete products) to allow your customer to select the variant they are looking for.
+Spryker supports an unlimited amount of super attributes for a given abstract product. See below to understand how abstract and concrete products works with super attributes.
 
-Consider the example of a T-shirt, the Abstract Product, that is available in the sizes small, medium, and large, three different Product Variants. The Abstract Product "T-shirt" appears as a search result. On the product detail page the customer can select between the product variants small, medium, large and put it into the cart.
+## Hierarchy of products
 
-![image](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Product+Abstraction/product-abstraction.png){height="" width=""}
\ No newline at end of file
+In Spryker, you have 2 product levels:
+* Abstract
+* Concrete
+
+An abstract product can contain 1 or more concrete products. A concrete product is only linked to 1 abstract product.
+
+### Differences between the levels
+
+Abstract product is the level where all the information about the product are located such as title, description, images, prices, brand.
+Concrete product is the level where all divergent information compared to the abstract product are located such as size, color, images, prices. It is the level where the stock for the product is defined. It is the only level where super attributes must be defined.
+
+### Inheritance between levels
+
+Whenever Spryker must display some information about a concrete product, it will execute the following logic.
+* Does the information exists on the concrete product level?
+  * If yes, Spryker uses it.
+  * If no, Spryker will look if the information is present on the abstract product level.
+* Does the information exists on the abstract product level?
+  * If yes, Spryker uses it.
+  * If no, then nothing is displayed.
+
+## Use cases
+
+### Case number 1
+
+If you sell books, most of the time, they do not have variations. So you will structure it in Spryker as 1 abstract product and 1 concrete product.
+The abstract product will contain all information about the product. The concrete product will only hold the stock information.
+
+### Case number 2
+
+You sell a product in 2 colors (blue and green). You will structure the product in Spryker as 1 abstract product and 2 concrete products. The green variant is more expensive than the blue one. Here you must use a super attribute to allow Spryker to offer a switch to your customer, so that they can select which variant they want to buy.
```

Comment:
I think this is bad example.

Originally we keep in a single abstract variations, which looks the same, like iPhones:
Iphone XGold - is an abstract.
Iphone X gold 16/32/64 -  3 concretes

Ther might be product group `iphone X`, grouping all the colors.

Reply:
Product group is something different it should not be mixed with the super attribute documentation. I can put a 3rd example with more crazy setup like variant by color, size and material